save-cli/src/nativeMain/kotlin/org/cqfn/save/cli/SaveConfigReader.kt

```diff
@@ -132,7 +133,7 @@ fun readPropertiesFile(propertiesFileName: String?): SaveProperties {
 }
 
 private fun tryToUpdateDebugLevel(properties: SaveProperties) {
-    logType = properties.logType ?: LogType.WARN
+    logType = GenericAtomicReference(properties.logType ?: LogType.WARN)
```

Comment:
Should be `logType.value = properties.logType ?: LogType.WARN`, otherwise you lose all advantages of atomics